common.md

```diff
@@ -275,11 +276,17 @@ The semantics accept timestamps in one of two formats:
 1. An ISO-8601 string.
 2. A unix timestamp
 
+We also include a (partial) internal timestamp validation check.
+
 ```k
   syntax Int ::= #ISO2Epoch(String) [function, hook(TIME.ISO2Epoch)]
   syntax Timestamp ::= #ParseTimestamp(String) [function]
   rule #ParseTimestamp(S) => #Timestamp(#ISO2Epoch(S)) requires findString(S, "Z", 0) >=Int 0
   rule #ParseTimestamp(S) => #Timestamp(String2Int(S)) requires findString(S, "Z", 0) <Int 0
+
+  syntax Bool ::= #IsLegalTimestamp(Timestamp) [function, functional]
+  // ----------------------------------------------------------------
+  rule #IsLegalTimestamp(#Timestamp(I:Int)) => I >=Int 0
```

Comment:
Should have an `owise` case if we want it to be `functional`, but not sure how to deal with unevaluated functions. Can `#ISO2Epoch` be marked `functional`? Maybe return `-1` for bad values?

Reply:
Should be OK, `#Timestamp` is the only constructor.